src/components/footer/components/FooterSocialItem.vue

```diff
@@ -88,20 +88,36 @@ export default {
                     border-color: $color-facebook;
                 }
             }
+
             &.instagram {
                 &:hover {
                     background: $color-instagram;
                     box-shadow: 0 0 0 2px $color-instagram;
                     border-color: $color-instagram;
                 }
             }
+
             &.twitter {
```

Comment:
Can the old twitter could be removed?

Reply:
I've removed all of the old twitter icon styles